### PR TITLE
Update development and tests related files (incl. Semaphore setup)

### PR DESCRIPTION
**Pull request by domininik**
See commit messages for details.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.6.3
```
Comment (Djurredejong) [inline .ruby-version:1]: This is probably just my ignorance: but what does this mean if you are including that gem in an app that runs for example ruby 2.4?

**Review by Djurredejong — APPROVED** [review]
All changes look fine. The commit messages very clearly explain what you have been doing. But it's not always clear (to me) why you did it.

**Review by stephan-roolvink — APPROVED** [review]

File: .ruby-version (hunk 1 above)
Comment (domininik) [reply .ruby-version:1]: This file is just for a gem development. So all gem tests are also running with this version. It does not force Ruby version which app may use.

**Comment by domininik** [comment]
Released as `1.1.0`

**Pull request merged**